Common/Source/Devices/devLX_EOS_ERA.cpp

```diff
@@ -1983,15 +1984,11 @@ TCHAR szName[MAX_VAL_STR_LEN];
   {                                  
     int rwdir = 0; 
```

Comment:
always "0" ...

Reply:
... that is correct in case it is no landaable
changed it in next commit for lanables